README.md

```diff
@@ -4,38 +4,68 @@ FWL (pronounced "fuel") are Fewlines Web Libraries.
 
 They are an ensemble of packages that allow us to build backend services in Typescript.
 
-## Database
+## Packages
+
+### Database
 
 Provides an interface for a Postgres database.
 
 [Database documentation](./packages/database/README.md)
 
-## Logging
+### Logging
 
 Provides an interface for a simple logger.
 
 [Logging documentation](./packages/logging/README.md)
 
-## Migration
+### Migration
 
 Provides a migration tool using SQL files and a CLI.
 
 [Migration documentation](./migration/README.adoc)
 
-## Tracing
+### Tracing
 
 Provides an interface over OpenTelemetry.
 
 [Tracing documentation](./packages/tracing/README.md)
 
-## Web
+### Web
 
 Provides a way of creating an Express application with custom Router and Handlers.
 
 [Web documentation](./packages/web/README.md)
 
-## OAuth2
+### OAuth2
 
 Provides a client that abstract Connect OAuth2 implementation, and gives tools to decrypt and verify JWS/JWE.
 
 [Web documentation](./packages/oauth2/README.md)
+
+
+## Publishing a new version of packages
+
+We only publish packages from the `main` branch which means we will make a PR bumping versions before publishing.
+
+### Update packages versions
+
+```shell-session
+yarn lerna version --no-push --no-git-tag-version --exact
+```
+
+This command will ask you for what versions you want to bump each package.
+
+With these options, lerna will only do the bumps in the `version` field of each `package.json`. This is optional and you could bump a version
+
+We then need to update the `CHANGELOG.md` files for each of them.
```

Comment:
So it's just changing the version field in the `package.json` for one of our packages? 
Why is it a thing, since we need to do it in the `CHANGELOG.md`? 🤔 
I'm sorry but I don't get why we should use that instead of writing it.

Reply:
> It's to do it for multiple packages once? 🤔

It can yes, 

In fact, it should be able to update the changelog but I didn't yet understood how it works 😅 (or maybe it's only linked to releases, I don't know yet).

However, the two options I used `--no-push` and `--no-git-tag-version` are (I hope) temporary since they cancel the fact that Lerna is creating tags for us. I don't know how to use it correctly right now, since it tags commits that will be squashed by our PRs that will then not really exist. I know I seem confused, it's because I am and I'm still discovering.

But in any case, yes, right now, it just bumps the version and that's all, that's just a convenience for bumping them all at once 😅

Reply:
> I know I seem confused

It is not. Thank you for the answer. 👍